icons/square/48/gnome-usage.svg

```diff
@@ -1,26 +1,21 @@
-<svg width="48" height="48" version="1.1" viewBox="0 0 48 48.000001" xmlns="http://www.w3.org/2000/svg">
+<svg width="48" height="48" version="1.1" viewBox="0 0 48 48" xmlns="http://www.w3.org/2000/svg">
```

Comment:
This `<svg>` tag is weird compared to the other icons. Not sure why width and height attributed are here?